index.js

```diff
@@ -23,9 +23,7 @@ module.exports = function(app) {
         app.use(express.static(path.join(__dirname, 'app')));
 
         viewsDirectory += '/app/views';
-    }
-
-    if ('production' === env) {
+    } else {
```

Comment:
I'm not clear with the distinction between these two modes.

What happens when I do `grunt serve` ? Will the app be launched in dev mode ?
And with `npm start` you can launch it in prod mode ?